server/server.js

```diff
@@ -41,17 +57,17 @@ app.post('/api',
   sessionController.isLoggedIn,
   userController.getBoardIds,
   boardController.getBoards,
-  (req, res) => {
+  (_, res) => {
     res.status(200).json(res.locals.boards)
   })
 
 app.post(
   "/login",
   userController.verifyUser,
   sessionController.startSession,
-  cookieController.setSSIDCookie,
   (_, res) => {
     console.log("completing post request to '/login");
+    // TODO: This will need to send back board IDs for the front end
     res.sendStatus(200);
   }
```

Comment:
Maybe I'm recalling this wrong, but I think we at some point changed the login response to send the user data from res.locals instead of just a status 200

Reply:
Oh wait .... I think my timeline is wrong - this PR is a reflection of before that happened. My bad!